python/llm/example/GPU/QLoRA-FineTuning/qlora_finetuning.py

```diff
@@ -21,6 +21,7 @@
 from transformers import LlamaTokenizer
 import intel_extension_for_pytorch as ipex
```

Comment:
try to remove import intel_extension_for_pytorch as ipex

Reply:
> try to remove import intel_extension_for_pytorch as ipex

This still needs to be manually tested @qiyuangong

Reply:
ok, we will check the performance difference with qlora example manually.

Reply:
Using qiyuan' ipex pr, I check the performance with qlora example. Example can run normally and the performance is the same as before.